backend/backoffice/admin.py

```diff
@@ -0,0 +1,15 @@
+from django.contrib import admin
+
+from backoffice.models import TalkEntry
+
+
+@admin.register(TalkEntry)
+class TalkEntryAdmin(admin.ModelAdmin):
+    search_fields = ("name", "title")
+    ordering = ("name", "date", "title")
+    list_display = ("name", "date", "title")
+    list_display_links = ("name", "date", "title")

```

Comment:
As we plan to also display external links (like the `youtube_link`), I think it's better to not transform all fields into links. This will avoid confusion between links redirecting to the edition form and links redirecting elsewhere.

```suggestion
list_display_links = ("date",)
```

Reply:
Good point, it's done